tools/sidekick/config.json

```diff
@@ -96,6 +96,17 @@
       "includePaths": [ "**.docx**" ],
       "passConfig": true,
       "passReferrer": true
+    },
+    {
+      "containerId": "tools",
+      "id": "version-history",
+      "title": "Version History",
+      "environments": [ "edit" ],
+      "url": "tools/version-history?project=milo--adobecom",
+      "isPalette": true,
+      "passReferrer": true,
+      "paletteRect": "top: auto; bottom: 20px; left: 20px; height: 498px; width: 460px;",
+      "includePaths": [ "**.docx**" ]
```

Comment:
Can we support version history for excel files too?

Reply:
Excel support added.